### DISCUSSION TITLE
Can I configure different time zone for Kiali?

### DISCUSSION THREAD

**Discussion by jmueducn** [kiali/kiali#7614] [Q&A] [answered]
It seems that I can't configure the time zone of Kiali, it is always the browser/system time.
However, in Prometheus, there is a "Use local time" checkbox.
In Grafana, you can configure the time zone global/in dashboard.
So, I'm wondering whether Kiali has these features.

**Comment by jmueducn** [comment]
Also, I want to know can I set the "metrics_per_refresh" not "Last xxx", but "From xxx to xxx"?

**Reply by jmazzitelli** [reply]
There isn't a `from <date> to <date>` ... there is the Replay feature but I am not sure if that is what you are asking for: 

- https://kiali.io/docs/features/topology/#replay
- https://www.youtube.com/watch?v=CC_dl4zSZiU

**Reply by jmueducn** [reply]
Great help!

**Comment by jmazzitelli** [comment, marked as answer]
I don't know the answer about timezone - I am pretty sure the answer is no, there is no timezone selector in Kiali UI.

**Reply by jmueducn** [reply]
Thanks for your information!
